examples/hpc-c/hello-newlib-hpc/README.md

```diff
@@ -0,0 +1,21 @@
+# Minimal C program
+
+Just a bare metal C program that makes ude of the Newlib C runtime library.
+
+## Quickstart
+
+1. Build and install Newlib on a directory of your choice and paste the include path and library path the `INC_PATH` and `LIB_PATH` of the CMakeLists.txt, accordingly.
+2. Make sure you've got a compatible `riscv64-unknown-elf-gcc` available on $PATH. 
```

Comment:
What does compatible mean? If it refers to compiler build with `cmodel=medany` that should be mentioned more clearly.

Reply:
The word compatible is redundant here. Even if the toolchain has been built with a different code model, the cmake file specifies that this program needs to be build with `medany` code model.